package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
     "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "test": "jest __tests__/app.test.js",
```

Comment:
when you run `npm test` or the shorthand `npm t` you can also specify a file name, and it will look for that file (I believe using regex, so doesn't even have to be the full name, just the beginning).

Example: `npm t ap` would search for a file starting 'ap_____' and ending with '.test.js' - in this case finding `app.test.js`

I like the idea to update the script but we don't always only want to test app.test.js, we may also want to test other files or pass commands to jest like -v.